packages/compass-editor/src/ace/theme.ts

```diff
@@ -0,0 +1,142 @@
+import { palette } from '@mongodb-js/compass-components';
+
+// To trick vscode into formatting and autocompleting the string
+const css = String.raw;
+
+const mongodbAceThemeCssText = css`
+  .ace-mongodb.ace_editor {
+    background: ${palette.gray.light3};
+    color: ${palette.black};
+  }
+  .inline-editor.ace-mongodb.ace_editor {
+    background: transparent;
+  }
```

Comment:
That's basically the only other relevant difference between query bar editor styles and other editors that need to be actually part of the theme, everything else can be done without the need to load an additional theme